spec/models/alaveteli_pro/embargo_extension_spec.rb

```diff
@@ -23,7 +23,7 @@
   end
 
   it "requires an embargo" do
-    embargo_extension.embargo = nil
+    embargo_extension.embargo_id = nil
```

Comment:
I need to work out what's going on there - something in the way the patch works means that the relation isn't invalidated by assigning nil as the object, it needs the database lookup to break, it must have introduced an extra lookup